man/systemd.exec.xml

```diff
@@ -1457,6 +1451,29 @@
       </varlistentry>
 
       <varlistentry>
+        <term><varname>RestrictNamespaces=</varname></term>
```

Comment:
I think there's potential for semantic confusion here. The options says "Restrict", and "yes" is about restricting, but individual values are about allowing. Let's compare this with DHCP: one could say that `DHCP=no < DHCP=ipv4/6 < DHCP=yes`, but here "yes" is not a superset of options. I see the appeal of calling this "Restrict*", but maybe "AllowNamespaces=" would be better (with a default of "yes", and "no" meaning complete refusal).